crypto/crypto_test.go

```diff
@@ -116,44 +116,46 @@ func TestCreate2(t *testing.T) {
 }
 
 func TestValidateSignatureValues(t *testing.T) {
```

Comment:
Is there any other extra test to check the downcasting?

Reply:
No, I'll try to add some new tests.

Reply:
Whatever test I write, it is very similar to this, so I would say, this test should be enough.